core/tests/webdriverio_utils/ExplorationEditorMainTab.js

```diff
@@ -420,6 +420,12 @@ var ExplorationEditorMainTab = function() {
     };
   };
 
+  this.invisibilityOfSelfLoopWarningInResponse = async function() {
```

Comment:
Function names should start with verbs. Follow existing conventions.

Reply:
@seanlip, we can use checkInvisibilityOfSelfLoopWarningInResponse or waitForInvisibilityOfSelfLoopWarningInResponse.
I prefer the first one as it suits the test flow.